package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.1",
   "description": "A Javascript port of the PySyft library",
   "main": "lib/index.js",
+  "module": "lib/index.esm.js",
```

Comment:
Are there any packages we need to install in order to allow for ESM?  If not, then this is fine.  However... I would much rather this PR be closed and have it all combined into one bigger PR with #33 .  @ahylton19 ?

Reply:
Nope we already have them. And okay !